cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -220,6 +223,17 @@ def get_device_specification(self) -> Optional[v2.device_pb2.DeviceSpecification
         else:
             return None
 
+    def get_default_device_config_key(self) -> quantum.DeviceConfigKey:
```

Comment:
My guess is that users will care about which config they're using (e.g. what's in the sampler), and less about what the default value is. Since we're already supplying the default to the sampler without making the user load this value, they probably won't need this method. WDYT?

Reply:
+1, removed and its associated tests